include/wlr/types/wlr_xdg_shell_v6.h

```diff
@@ -152,6 +152,13 @@ struct wlr_xdg_toplevel_v6_resize_event {
 	uint32_t edges;
 };
 
+struct wlr_xdg_toplevel_v6_set_fullscreen_event {
+	struct wl_client *client;
```

Comment:
Why are we putting the client in these events? Shouldn't it be always the same as the client for the shell surface?

Reply:
I just copy-pasta-ed the boilerplate from other events. So yeah, I don't think the client is useful in these events. Let's remove it.